packages/forms-web-app/src/dynamic-forms/question.js

```diff
@@ -51,6 +61,8 @@ class Question {
 	url;
 	/** @type {string|undefined} optional html content */
 	html;
+	/** @type {Question|undefined} */
+	subQuestion;
```

Comment:
I'm not sure we need to support this at this level, at least not yet. The only use case I'm aware of so far is on the ListAddMore type questions where we want to add items to a list? Would it make it simpler if we moved this down to the level where it is currently needed or does it not make that much difference in reality?

Reply:
I need to know about it in validator.js
However I can probably check instance of is AddMore and then check subquestion
Not that it matters that much due to it being js, more a jsdoc thing

Reply:
So yes will move

Reply:
Good old jsdoc